### PR TITLE
Feat/apigw-ws

### LINKED ISSUE #8: Support WebSocket fields in requestContext
When using function with WebSocket integration there are several [additional fields](https://cloud.yandex.ru/docs/functions/concepts/function-invoke#request) in `event.requestContext`, e.g. `connectionId`, `eventType`, etc. It would be great to support these fields in typings.

### PR DESCRIPTION
**Pull request by esseswann**
Closes #8

### CODE REVIEW

File: examples/iotCore.ts [hunk 1]
```diff
@@ -3,6 +3,6 @@ import { Handler } from "../src";
 export const handler: Handler.IotCore = (event) => {
     return {
         statusCode: 200,
-        body: event.messages[0]?.details.mqtt_topic.toString() || '',
+        body: event.messages[0]?.details.mqtt_topic.toString() || "",
```
Comment (vitalets) [inline examples/iotCore.ts:6]: I think changing quotes style should be done in separate PR if really needed.

Comment (esseswann) [reply examples/iotCore.ts:6]: This is rather not a change in style, it's a harmonization since they were not consistent. Since this is not a breaking change I would prefer not extracting it to a separate request :)

File: src/index.ts [hunk 2]
```diff
@@ -13,10 +14,30 @@ import {
...
+                Http.Result
+            >;
+            export type Disconnect = FunctionHandler<
+                IWebSocket.Disconnect,
+                Http.Result
+            >;
+        }
+    }
+
+    // Backward compatibility
```
Comment (DavyJohnes) [inline src/index.ts:38]:
```suggestion
    // Backward compatibility
    // @deprecated use `ApiGateway.Authorizer` instead
```

File: src/api-gateway/websocket.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import type { Http } from "../http";
+
+export namespace WebSocket {
```
Comment (DavyJohnes) [inline src/api-gateway/websocket.ts:3]: А здесь то зачем неймспейс? Можно же по месту импорта сделать `* as WebSocket`.

File: src/api-gateway/websocket.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    export type Connect = Common<Headers> & {
```
Comment (DavyJohnes) [inline src/api-gateway/websocket.ts:4]: Для большей прозрачности при чтении я бы предложил явно дописать суффикс `Event`.

**Review by DavyJohnes — DISMISSED** [review]
